### DISCUSSION TITLE
Data retention and deletion policies

### DISCUSSION THREAD

**Discussion by eilonmore** [deepgram/community#324] [General help] [answered]
Hey there,

I have been reviewing the terms of use of Deepgram, and I had a few questions regarding data retention and deletion policies for unused transcriptions and audios. The terms of use does not explicitly state that it will delete the transcription data (audio and transcription) after processing it.

**Data Deletion Policy**: Could you please provide more information about Deepgram's policies for deleting unused transcriptions and audio data after they have been processed? Specifically, is there an automatic process that deletes this data after a certain period, or do we need to request deletion manually?

**Deletion Rules**: How can I set up rules to automatically delete past audio and conversation data after a specified period?

**Manual Deletion**: How can I manually delete past audio and conversation data from Deepgram's servers?

Any assistance would be greatly appreciated.

**Comment by lukeocodes** [comment, marked as answer]
Hey @eilonmore I'm closing this as a duplicate to #115 

If you have any concerns about your data and Deepgram, you're welcome to contact security@deepgram.com 

Thanks!
